docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  yt-chat:
+    image: yt-chat:latest
+    build: ./app
+    volumes:
+      - ./yt_chat/:/app/yt_chat
```

Comment:
Is there a particular reason for wanting to mount volumes? is this for handling downloads of Mistral weights?